portafly/package.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Can we make it a dependency of main package.json? I mean like bundler and our developer portal?
Asking about license finder and also how we install porta in general

Reply:
We could adapt the task to run it with that package.json, but Porta package.json should not be mixed with this one